lib/bin/bt_ll_acs_nrf53/src/ble_hci_vsc.c

```diff
@@ -10,7 +10,7 @@
 #include <zephyr/bluetooth/hci.h>
 
 #include <zephyr/logging/log.h>
-LOG_MODULE_DECLARE(ble, CONFIG_BLE_LOG_LEVEL);
+LOG_MODULE_REGISTER(ble_hci_vsc, 4);
```

Comment:
Discuss naming

Reply:
Discuss as this file is not part of the contents of this PR and resides within another repo.